GuildWars2-Calc/src/main/java/com/theironyard/services/GuildWar2CalcService.java

```diff
@@ -0,0 +1,52 @@
+package com.theironyard.services;
+
+import com.theironyard.entities.Item;
+import com.theironyard.entities.Price;
+import com.theironyard.entities.Recipe;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+
+/**
+ * Created by EddyJ on 7/25/16.
+ */
+@Service
+public class GuildWar2CalcService {
+
+    @Autowired
+    RestTemplate restTemplate;
+
+    public static final String RECIPE_URL = "https://api.guildwars2.com/v2/recipes/";
```

Comment:
Might be worth having a BASE_URL and then each of the urls just add the suffix to it.